src/state/pipeline-run-tasks/index.js

```diff
@@ -0,0 +1,7 @@
+import { makeLocalGetter } from '../../utils/object';
+import get from 'lodash/get';
+
+const localGetter = makeLocalGetter('pipelineRunTasks');
+
+export const getPipelineRunTasks = (state) =>
+  get(state, 'pipelineRunTasks', []);
```

Comment:
remove unused getter
```suggestion
import get from 'lodash/get';

export const getPipelineRunTasks = (state) =>
  get(state, 'pipelineRunTasks', []);
```